docs/content/en/contribute/releasing/_index.md

```diff
@@ -206,7 +211,7 @@ git cherry-pick -n <your commit SHA-1s...>
 Amend the merge commit with your cherry-picks:
 ```shell
 git add .
-git commit --amend
+git commit --amend --no-edit
```

Comment:
nitpick: Not my cup of tea, I like to have the ability to edit the commit message when amending since it might be out of date. Thus, --no-edit feels odd.

Reply:
It's a merge commit though, I don't think we've ever had the need to reword it. [Example](https://github.com/k8ssandra/k8ssandra-operator/commit/3dd3c6b702d54e37c7527f060ebf25fa41d93805)

I'm going to leave it as is since it's the most common case, it's easy enough to modify the command or do another `commit --amend` if you want to change the message.